src/components/list-box/_list-box.scss

```diff
@@ -0,0 +1,242 @@
+    transform: rotate(180deg);
+  }
+
+  .bx--list-box__selection {
+    display: inline-block;
+    position: absolute;
+    top: 0;
+    right: rem(26px);
+    bottom: 0;
+    height: rem(40px);
+    padding: 0 1rem;
+    cursor: pointer;
+    user-select: none;
+    transition: background-color 200ms $bx--standard-easing;
+  }
+
+  .bx--list-box__selection > svg {
+    fill: $ui-05;
+    height: 100%;
+  }
+
+  .bx--list-box__selection:focus {
+    outline: 1px solid $brand-01;
+  }
+
+  .bx--list-box__selection--multi {
+    position: static;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+    background-color: $brand-01;
+    height: rem(18px);
+    color: white;
+    font-size: 12px;
+    line-height: 0;
+    padding: rem(5px);
+    margin-right: rem(10px);
+    border-radius: rem(13px);
+  }
+
+  .bx--list-box__selection--multi > svg {
+    fill: white;
+    width: 0.5rem;
+    height: 0.5rem;
+    margin-left: rem(5px);
+  }
+
+  .bx--list-box__selection--multi:focus,
+  .bx--list-box__selection--multi:hover {
+    background-color: $brand-02;
+    outline: none;
+  }
+
+  .bx--list-box__field {
+    @include button-reset;
+    position: relative;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+    padding: 0 1rem;
+    cursor: pointer;
+  }
+
+  .bx--list-box__field:focus,
+  .bx--list-box__field[aria-expanded='true'] {
+    outline: 1px solid $brand-01;
+  }
+
+  .bx--list-box__field[disabled] {
+    outline: none;
+  }
+
+  .bx--list-box__field[aria-expanded='true'] ~ .bx--list-box__menu {
+    box-shadow: 0 1px 0 0 $brand-01, 1px 0 0 0 $brand-01, -1px 0 0 0 $brand-01,
+      0 4px 8px 0 rgba(0, 0, 0, 0.1);
+  }
+
+  .bx--combo-box > .bx--list-box__field {
+    padding: 0;
+  }
+
+  .bx--list-box__label {
+    color: $text-01;
+    font-size: 14px;
+    font-weight: 600;
+    user-select: none;
+  }
+
+  .bx--list-box__menu-item:hover,
+  .bx--list-box__menu-item--highlighted {
+    background-color: rgba($field-01, 0.1);
+    color: $text-01;
+  }
+
+  .bx--list-box--inline {
+    background-color: inherit;
+    width: auto;
+    height: $list-box-inline-height;
+  }
+
+  .bx--list-box--inline .bx--list-box__label {
+    color: $brand-01;
+  }
+
+  .bx--list-box--inline .bx--list-box__field {
+    padding: 0 0 0 rem(10px);
+  }
+
+  .bx--list-box--inline .bx--list-box__menu-icon {
+    position: static;
+    padding: 0 rem(10px) 0 rem(12px);
+  }
+
+  .bx--list-box--inline > .bx--list-box__menu {
+    top: $list-box-inline-height;
+    width: $list-box-width;
+  }
+
+  .bx--list-box--inline
+    > .bx--list-box__field[aria-expanded='true']
+    ~ .bx--list-box__menu {
+    outline: 1px solid $brand-01;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.1);
+  }
+
+  .bx--list-box--inline > .bx--list-box__field[aria-expanded='true'] {
+    outline: none;
+  }
+
+  .bx--dropdown-v2.bx--list-box--inline .bx--list-box__selection {
```

Comment:
I have a feeling I got lazy towards the end here and started just adding in custom selectors for slight problems that I had, visually speaking, when working directly on: https://github.com/carbon-design-system/carbon-components-react/pull/332

Reply:
I think this makes sense, I've seen way worse in our repo 😂